.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
                 '',
                 {
                     pattern: '^ \\* Copyright \\(c\\) \\d{4}, (salesforce.com, inc|Salesforce, Inc)\\.$',
-                    template: ' * Copyright (c) 2022, Salesforce, Inc.'
```

Comment:
@wjhsf since this is a JS file, can we use JS here to always put in the correct year?

Reply:
🧠